users/pdl/pdl.h

```diff
@@ -0,0 +1,115 @@
+#pragma once
+
+#include "quantum.h"
+
+enum userspace_layer_codes {
+    QWERTY = SAFE_RANGE,
+    PROXIM,
+    PUNCTN,
+    CODING,
+    KU_LBRC,
+    KU_NUHS,
+    KU_GRV,
+    KU_RBRC,
+    NEW_SAFE_RANGE
+};
+
+#define _QWERTY 2
+#define _PROXIM 0
+#define _NUMBRS 4
+#define _NAVIGN 5
+#define _PUNCTN 6
+#define _CODING 7
+#define _FUNCTN 8
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```